### PR TITLE
KEYCLOAK-12483 Add Arabic localization

### PR DESCRIPTION
**Pull request by mvk375**

### CODE REVIEW

**Comment by stianst** [comment]
Thanks for the PR. We need someone from the community to review this so sent an email asking for volunteers on dev and user mailing lists.

File: themes/src/main/resources-community/theme/base/account/messages/messages_ar.properties [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+requestsWaitingApproval=طلباتك في انتظار الموافقة
+icon=أيقونة
+requestor=الطالب
+owner=صاحب
+resourcesSharedWithMe=الموارد المشتركة معي
+permissionRequestion=طلبات إذن
+permission=الإذن
+shares=شارك
+
+locale_ca=Catal\u00e0
```
Comment (stianst) [inline themes/src/main/resources-community/theme/base/account/messages/messages_ar.properties]: This is mixing encoding. The locale_* properties doesn't need to be added here as they can be inherited from messages_en.properties instead.

**Review by pkumar76 — APPROVED** [review]
Changes looked good. verified the changes in local build.

File: themes/src/main/resources-community/theme/base/account/messages/messages_ar.properties (hunk 1 above)
Comment (mvk375) [reply themes/src/main/resources-community/theme/base/account/messages/messages_ar.properties]: Thank you

File: themes/src/main/resources-community/theme/base/login/messages/messages_ar.properties [hunk 2]
```diff
@@ -0,0 +1,261 @@
+#encoding: utf-8
+doLogIn=سجل الان
```
Comment (malmuzaini) [inline themes/src/main/resources-community/theme/base/login/messages/messages_ar.properties]: `الآن`

File: themes/src/main/resources-community/theme/base/login/messages/messages_ar.properties [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+doRegister=ﺳﺠﻞ اﻻن‬
+doCancel=‫إﻟﻐﺎء‬
+doSubmit=‫ﺧﻀﻊ‬
```
Comment (malmuzaini) [inline themes/src/main/resources-community/theme/base/login/messages/messages_ar.properties]: إرسال

File: themes/src/main/resources-community/theme/base/login/messages/messages_ar.properties [hunk 4]
```diff
@@ -0,0 +1,261 @@
...
+doYes=ﻧﻌﻢ ﻓﻌﻼ‬
```
Comment (malmuzaini) [inline themes/src/main/resources-community/theme/base/login/messages/messages_ar.properties]: `نعم` كافية

File: themes/src/main/resources-community/theme/base/login/messages/messages_ar.properties [hunk 5]
```diff
@@ -0,0 +1,261 @@
...
+doNo=‫ﻻ‬
+doContinue=‫اﺳﺘﻤﺮ‬
```
Comment (malmuzaini) [inline themes/src/main/resources-community/theme/base/login/messages/messages_ar.properties]: متابعة

File: themes/src/main/resources-community/theme/base/login/messages/messages_ar.properties [hunk 6]
```diff
@@ -0,0 +1,261 @@
...
+doAccept=‫ﻗﺒﻮل‬
+doDecline=‫اﻧﺨﻔﺎض‬
```
Comment (malmuzaini) [inline themes/src/main/resources-community/theme/base/login/messages/messages_ar.properties]: رفض

**Comment by malmuzaini** [comment]
Please don't merge it the translation need to be enhanced

**Comment by malmuzaini** [comment]
is it a google translation?

File: themes/src/main/resources-community/theme/base/account/messages/messages_ar.properties [hunk 7]
```diff
@@ -0,0 +1,352 @@
+#encoding: utf-8
+doSave=حفظ
+doCancel=إلغاء
+doLogOutAllSessions=تسجيل الخروج من جميع جلسات
+doRemove=أزل
+doAdd=أضف
+doSignOut=خروج
```
Comment (malmuzaini) [inline themes/src/main/resources-community/theme/base/account/messages/messages_ar.properties]: تسجيل خروج

File: themes/src/main/resources-community/theme/base/account/messages/messages_ar.properties [hunk 8]
```diff
@@ -0,0 +1,352 @@
...
+doLogIn=تسجيل الدخول
+doLink=حلقة الوصل
```
Comment (malmuzaini) [inline themes/src/main/resources-community/theme/base/account/messages/messages_ar.properties]: ربط

**Comment by malmuzaini** [comment]
Can I submit a commit with the modifications  ? Most of it is totally wrong! Is it google translation ?

**Comment by stianst** [comment]
@mvk375 any response to @malmuzaini review?

I'm not able to review the PR at all so can't really give any input here, but it seems a bit odd to me that this is approved by @pkumar76, while @malmuzaini believes it's completely wrong.

**Comment by HabooshHaddad** [comment]
I had a look at the translated texts and I agree with Mashail ..
needs enhancements .. looks like its translated by google !

**Comment by stianst** [comment]
Thanks @mvk375 and @HabooshHaddad. I'm going to close this PR then. If anyone wants to send a proper translation we would welcome that.

**Pull request closed without merging**